app.js

```diff
@@ -75,8 +75,8 @@ app.use((err, req, res, next) => {
   }
 
   if (err.response) {
-    // extract error message from V3 API
-    errorResponse.message = _.get(err, 'response.body.result.content')
+    // extract error message from V3/V5 API
+    errorResponse.message = _.get(err, 'response.body.result.content') || _.get(err, 'response.body.message')
```

Comment:
I made a slight improvement here by the way.

Reply:
yeah, this change looks good